readme.md

```diff
@@ -31,32 +33,76 @@ Don't see it? Try reinstalling Package Control. Altenatively, add the repository
 
 If it doesn't work, please file an issue.
 
+If you want contribute or use the latest commits simply git clone the repository `https://github.com/mrjoelkemp/Dependents` to your Packages directory with a directory name of `Dependents`. From sublime choose `Preferences -> Browse Packages`  to easily determine the path to the Packages directory for your install.  Be sure to remove the package via package control first if you had already done so to avoid package control updating the package.
+
+### Configuring settings
+
+There are two ways in which to invoke the package settings.  
+
+One is globally via entires in a user settings file `Preferences -> Package Settings -> Dependents -> Settings-User`.  You can open the `Settings-Default` file and copy and paste the contents as a starter template.   Do not edit the default file as that will be overwritten with updates.
```

Comment:
From my perspective just remove it now.  It makes little sense and is of
little use offering the global option and only makes for more confusion.
Related would be to remove the menu item preferences->packages
settings->dependents so folks aren't tempted.

On Wed, Mar 11, 2015 at 7:21 PM, Joel Kemp <notifications@github.com> wrote:

> In readme.md
> <https://github.com/mrjoelkemp/Dependents/pull/131#discussion_r26274021>:
>
> > @@ -31,32 +33,76 @@ Don't see it? Try reinstalling Package Control. Altenatively, add the repository
> >
> >  If it doesn't work, please file an issue.
> >
> > +If you want contribute or use the latest commits simply git clone the repository `https://github.com/mrjoelkemp/Dependents` <https://github.com/mrjoelkemp/Dependents> to your Packages directory with a directory name of `Dependents`. From sublime choose `Preferences -> Browse Packages`  to easily determine the path to the Packages directory for your install.  Be sure to remove the package via package control first if you had already done so to avoid package control updating the package.
> > +
> > +### Configuring settings
> > +
> > +There are two ways in which to invoke the package settings.
> > +
> > +One is globally via entires in a user settings file `Preferences -> Package Settings -> Dependents -> Settings-User`.  You can open the `Settings-Default` file and copy and paste the contents as a starter template.   Do not edit the default file as that will be overwritten with updates.
>
> I wanted to phase out the global settings from the readme because unless a
> dev uses the same structure for every project, this locks the editor into
> the configuration of a single project. The .deprc solution is
> significantly more flexible.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mrjoelkemp/Dependents/pull/131/files#r26274021>.
>



-- 
Cheers, Cuidate, TTFN <http://www.youtube.com/watch?v=5Gu50vq5ux4>
David